dataactcore/config_example.yml

```diff
@@ -98,86 +157,82 @@ services:
+    # host: broker-sbx.cvhpn1mlakcs.us-gov-west-1.rds.amazonaws.com
+    host: dataact-postgres
     port: 5432
 
+    # Set your username and password for the db instance
+    username: admin
+    password: root
+
     # Set the names for each of the application databases
-    base_db_name: postgres #This is the default db on the instance. 
+    base_db_name: postgres #This is the default db on the instance.
+    user_db_name: user_manager # User manager db
+    job_db_name: job_tracker # Job tracker db
+    staging_db_name: staging # Staging db
+    validator_db_name: validation # Validation db
+    error_db_name: error_data # Error data db
+    job_queue_db_name: job_queue # Job queue db
     db_name: data_broker
 
+    # The broker uses DynamoDb for session management. If
+    # use_aws is true above, the broker will use a
+    # Dynamo instance on your AWS account. Otherwise,
+    # provide a dynamo host and port below.
+    dynamo_host: 127.0.0.1
+    dynamo_port: 5555
+
 logging:
 
     # The path where broker still store log files.
-    log_files: /full/path/to/logfiles
-
-    values_to_log:
-        broker:
-            - path
-            - use_aws
-            - local
-            - aws_bucket
-            - certified_bucket
-            - aws_role
-            - aws_region
-            - aws_create_temp_credentials
-            - static_files_bucket
-            - help_files_path
-            - d_file_storage_path
-            - broker_files
-            - award_procurement_url
-            - award_procurement_file_name
-            - award_url
-            - award_file_name
-            - awardee_attributes_url
-            - awardee_attributes_file_name
-            - executive_compensation_url
-            - executive_compensation_file_name
-            - sub_award_url
-            - sub_award_file_name
-            - rss_folder
-            - rss_file
-            - generated_files_folder
-            - sf_133_folder
-            - sf_133_file
-            - full_url
-        services:
-            - error_report_path
-            - broker_api_host
-            - validator_host
-            - broker_api_port
-            - validator_port
-            - protocol
-            - rest_trace
-            - server_debug
-        db:
-            - base_db_name
-            - scheme
-        logging:
-            - log_files
+    # Ignored if use_logstash is true.
+    log_files: /data-act/backend/data-act-broker-backend/temp/logs
+    # Set to true and fill in all logstash_ settings
+    # to use logstash for logging
+    use_logstash: false
+
+    # Logstash EC2 host. Ignored if use_logstash is false.
+    # NOTE: the dummy value below MUST be changed to the correct
+    # value if use_logstash is true.
+    logstash_host: logstash-host.compute-1.amazonaws.com
+
+    # Logstash EC2 port. 514 is the port typically used with logstash,
+    # so you can mostly likely keep this as is.
+    logstash_port: 514
+
+job-queue:
```

Comment:
I have a strange feeling we don't use this at all anymore (we don't have to run rabbit personally anymore, that was something we had to do before). Double-check, see if everything runs without it, if so gut it. Otherwise, I'd say at least remove the real URL and uncomment the port one.